webpack.config.js

```diff
@@ -64,15 +64,8 @@ var config = {
       },
       {
         test: FONT_REGEX,
-        loader: 'url',
-        query: {
-          // limit: 1000 // 10kb
-        }
+        loader: 'file?publicPath=/dist/'
```

Comment:
We can't have absolute paths for assets. publicPath query is interesting. Maybe it works if value is just `dist/`. I have to try.